helix-term/src/commands/typed.rs

```diff
@@ -459,10 +459,14 @@ fn format(
     if let Some(format) = doc.format() {
         let callback = make_format_callback(doc.id(), doc.version(), view.id, format, None);
         cx.jobs.callback(callback);
+    } else {
+        cx.editor
+            .set_error("Formater not available and no LSP has formatting capabilities");
```

Comment:
For typable commands you can return an `Err` for the same effect as `cx.editor.set_error`:

```suggestion
        bail!("Formatter not available and no language server has formatting capabilities");
```

Reply:
I didn't know that I can also just return the error, thank you for this suggestion!